test/Microsoft.AspNetCore.SignalR.Tests.Utils/ServerFixture.cs

```diff
@@ -144,65 +144,66 @@ public ILogger CreateLogger(string categoryName)
                 return _loggerFactory.CreateLogger(categoryName);
             }
         }
+    }
+
+    // TestSink doesn't seem to be thread-safe :(.
```

Comment:
It is now https://github.com/aspnet/Logging/commit/069892b6940050101ad569ba5806138b6d325d21

Reply:
This is test code so no rush before RC